db/migrate/20190527231403_add_status_and_remove_completed_from_rebate_form.rb

```diff
@@ -0,0 +1,29 @@
+class AddStatusAndRemoveCompletedFromRebateForm < ActiveRecord::Migration[5.2]
+  def up
+    add_column :rebate_forms, :status, :string, default: 'not signed'
+    update_existing_rebate_forms_up
+    remove_column :rebate_forms, :completed
```

Comment:
Are there any existing applications we need to migrate here? I had a mixture of completed and not completed applications, but now they're all not signed.
If we run this migration on UAT all applications will be considered not signed, even though they do have signatures.

Reply:
@Br3nda did you see the code for `update_existing_forms_up'? It should update all existing forms, and on my local it is respects the mix of completed and not.

Reply:
Worth considering - on both my and Cara's machines this migration does preserve that status as intended. Current understanding is there is no data on UAT which must be preserved but lets check with @robholmesnz before we deploy

Reply:
I'm rolling back and retesting the migration

Reply:
My mistake -- they all migrated BEAUTIFULLY  :rose:

Reply:
@Br3nda Oh good, that's a relief! 🎉